spec/facility_spec.rb

```diff
@@ -23,4 +25,229 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+  
+  describe '#offer services' do
+    it 'exists with more services' do
+      facility_1 = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014' })
+      facility_2 = Facility.new({name: 'Ashland DMV Office', address: '600 Tolman Creek Rd Ashland OR 97520', phone: '541-776-6092' })
+    
+      expect(facility_1).to be_a(Facility)
+      expect(facility_2).to be_a(Facility)
+    end
+
+    it 'can register a vehicle' do
+      facility_1 = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014' })
+      facility_2 = Facility.new({name: 'Ashland DMV Office', address: '600 Tolman Creek Rd Ashland OR 97520', phone: '541-776-6092' })
+      cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+      bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+      camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice} )
+      
+      expect(cruz).to be_a(Vehicle)
+      expect(bolt).to be_a(Vehicle)
+      expect(camaro).to be_a(Vehicle)
+    end
+    
+    it 'adds a new service "vehicle registration"' do
+      facility_1 = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014' })
+      
+      expect(facility_1.add_service('Vehicle Registration')).to eq(["Vehicle Registration"])
+      expect(facility_1.services).to eq(['Vehicle Registration'])
+    end
+    
+    it 'can collect fees' do
```

Comment:
So in order to test this more robustly, you would need to start with your `collected_fees` as zero, then register a vehicle and check `collected_fees` again.